BCStudentSoftwareDevTeam/celts
# 117 volunteer status

Task: On the eventList Page when you click on the Volunteer Details tab, the volunteer groups(attended, waitlisted, or RSVPed) can be checked and a big list of people will be shown from either of the three groups which is hard to tell who RSVPed, attended, or waitlisted. The task here is to add a new column on a display table that specifies what specific volunteer group a volunteer belongs to, whether attended, waitlisted, or RSVPED.

test: 

1. Go to the Event List page 
2. select the event you want to view 
3. Then click on the Volunteer Details Tab
4.  Check the Volunteer Status checkbox
5. Make sure the Event status column/Line is displayed on the Table View or the volunteer status is displayed on the card View

resolves #1117
